source/transactions/transactions.rst

```diff
@@ -160,6 +160,8 @@ This section is an overview of the public API for transactions:
         /**
          * The maximum amount of time to allow a single commitTransaction
          * command to run.
+         *
+         * NOTE: This option is deprecated in favor of timeoutMS.
```

Comment:
`timeoutMS` specified where?

Reply:
AFAIK, I can't link to other specs in code comments, so the existing `maxCommitTimeMS` section below also includes a deprecation notice with a link to the `timeoutMS` section of the CSOT spec.

Reply:
Sorry, I meant specified by the application, not the specification

Reply:
As mentioned in a previous comment, I added a `WithTransactionOptions` type to hold this.